### PR TITLE
Termui fixes april release

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
- don't display the status as synchronized if termui just started
- display redundancy info (main machine or back-up number + is the main machine is active)
- small fixes after late review for esdt supply endpoint

### CODE REVIEW

**Review by raduchis — DISMISSED** [review]

File: cmd/termui/view/termuic/termuiRenders/widgetsRender.go [hunk 1]
```diff
@@ -263,6 +269,26 @@ func (wr *WidgetsRender) prepareChainInfo(numMillisecondsRefreshTime int) {
...
+	if redundancyIsMainActive == statusNotApplicable {
+		return ""
+	}
+
+	redundancyStr := "Redundancy: "
+	if redundancyLevel < 0 {
+		redundancyStr += "inactive"
+	} else {
+		if redundancyLevel == 0 {
+			redundancyStr += "Main machine"
```
Comment (AdoAdoAdo) [inline cmd/termui/view/termuic/termuiRenders/widgetsRender.go:282]: lower case first letter? to be similar to `back-up ...`

**Review by AdoAdoAdo — APPROVED** [review]

File: cmd/termui/view/termuic/termuiRenders/widgetsRender.go [hunk 2]
```diff
@@ -162,6 +165,8 @@ func (wr *WidgetsRender) prepareInstanceInfo() {
 	countAcceptedBlocks := wr.presenter.GetCountAcceptedBlocks()
 	rows[4] = []string{fmt.Sprintf("Blocks proposed: %d | Blocks accepted:  %d", countLeader, countAcceptedBlocks)}
 
+	rows[5] = []string{computeRedundancyStr(wr.presenter.GetRedundancyLevel(), wr.presenter.GetRedundancyIsMainActive())}
+
 	// TODO: repair the rewards estimation or replace these 2 rows with rating details
```
Comment (iulianpascalau) [inline cmd/termui/view/termuic/termuiRenders/widgetsRender.go:170]: I do not think this todo is still relevant and the code can be removed. Rewards are visible on the explorer

File: cmd/termui/view/termuic/termuiRenders/widgetsRender.go [hunk 3]
```diff
@@ -211,6 +215,8 @@ func (wr *WidgetsRender) prepareChainInfo(numMillisecondsRefreshTime int) {
 
 		blocksPerSecond := wr.presenter.CalculateSynchronizationSpeed(numMillisecondsRefreshTime)
 		blocksPerSecondMessage = fmt.Sprintf("%d blocks/sec", blocksPerSecond)
+	case synchronizedRound == currentRound && currentRound == 0:
```
[the remark this answers is not in the document]
Comment (bogdan-rosianu) [reply cmd/termui/view/termuic/termuiRenders/widgetsRender.go:218]: right. fixed

File: cmd/termui/view/termuic/termuiRenders/widgetsRender.go (hunk 1 above)
Comment (bogdan-rosianu) [reply cmd/termui/view/termuic/termuiRenders/widgetsRender.go:282]: fixed

File: cmd/termui/view/termuic/termuiRenders/widgetsRender.go (hunk 2 above)
Comment (bogdan-rosianu) [reply cmd/termui/view/termuic/termuiRenders/widgetsRender.go:170]: right. removed

**Review by raduchis — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by popenta — COMMENTED** [review]
@@                            Log scanner                                  @@
##                            termui-fixes-april-release                   ##
================================================================================
- Known Warnings              1001
- New Warnings                480
- Known Errors                2
- New Errors                  2
- Panics                      0
================================================================================

**Pull request merged**